release/win/samples/conf/database-conf.xml

```diff
@@ -2,44 +2,46 @@
 <beans>
 <bean id="dbDataSource"
       class="org.apache.commons.dbcp.BasicDataSource"
-      destroy-method="close">
+      destroy-method="close"
+      scope="prototype">
```

Comment:
Let's make sure these changes do not have any regressions.

Reply:
agree.